client/src/main/java/org/asynchttpclient/netty/NettyResponseFuture.java

```diff
@@ -49,8 +48,8 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(NettyResponseFuture.class);
 
-    private static final AtomicIntegerFieldUpdater<NettyResponseFuture<?>> REDIRECT_COUNT_UPDATER = newAtomicIntegerFieldUpdater(NettyResponseFuture.class, "redirectCount");
-    private static final AtomicIntegerFieldUpdater<NettyResponseFuture<?>> CURRENT_RETRY_UPDATER = newAtomicIntegerFieldUpdater(NettyResponseFuture.class, "currentRetry");
+    private static final AtomicIntegerFieldUpdater<NettyResponseFuture> REDIRECT_COUNT_UPDATER = AtomicIntegerFieldUpdater.newUpdater(NettyResponseFuture.class, "redirectCount");
+    private static final AtomicIntegerFieldUpdater<NettyResponseFuture> CURRENT_RETRY_UPDATER = AtomicIntegerFieldUpdater.newUpdater(NettyResponseFuture.class, "currentRetry");
```

Comment:
Error:(51, 137) java: incompatible types: inferred type does not conform to equality constraint(s)
    inferred: org.asynchttpclient.netty.NettyResponseFuture<?>
    equality constraints(s): org.asynchttpclient.netty.NettyResponseFuture<?>,org.asynchttpclient.netty.NettyResponseFuture